### PR TITLE
Removed tkinter

### LINKED ISSUE #4799: Drop Tkinter
At some point in time `tkinter` was shipped with Python.

However, this is no longer the case. 

Therefore we should not use `tkinter` anymore as a fallback option and switch to some other library we already use anyway (like Qt).

### PR DESCRIPTION
**Pull request by qstokkink**
Fixes #4799

### CODE REVIEW

**Comment by qstokkink** [comment]
retest this please

File: check_os.py [hunk 1]
```diff
@@ -17,53 +17,30 @@
...
+    Create a native pop-up without any third party dependency.
+
+    :param title: the pop-up title
+    :param text: the pop-up body
+    """
+    try:
+        import win32api
+        win32api.MessageBox(0, text, title)
+    except ImportError:
+        os.system('notify-send -u "critical" "%s" "$(printf "%s")"' % (title, text))
```
Comment (devos50) [inline check_os.py]: `notify-send` is not available on MacOS :(

File: check_os.py [hunk 2]
```diff
@@ -17,53 +17,30 @@
...
+    Create a native pop-up without any third party dependency.
+
+    :param title: the pop-up title
+    :param text: the pop-up body
+    """
+    try:
+        import win32api
+        win32api.MessageBox(0, text, title)
+    except ImportError:
+        os.system('xmessage -center "$(printf "%s")"' % (text))
```
Comment (arvidn) [inline check_os.py]: I would like to point out that this appears to be a security risk. At least a ticking time-bomb. If an attacker can control the content of `text`, any arbitrary shell command can be executed.

Comment (arvidn) [reply check_os.py]: using `os.subprocess` with `shell=False` is probably sufficient to fix it btw

Comment (ichorid) [reply check_os.py]: I confirm that using `os.system` is a security risk and is considered to be a bad practice these days. `os.subprocess` with `shell=False` is the way to go safely.

Comment (devos50) [reply check_os.py]: Thanks for your feedback! I will change this as part of my PR (#4815)

**Pull request merged**